var/spack/repos/builtin/packages/py-colorspacious/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyColorspacious(PythonPackage):
+    """`colorlog.ColoredFormatter` is a formatter for use with Python's
+    `logging` module that outputs records using terminal colors.
+     """
+
+    homepage = "https://github.com/borntyping/python-colorlog"
```

Comment:
@Bidibulke The link in my previous comment comes from this package :slightly_smiling_face:

Reply:
Oups. I didn't see that.